### PR TITLE
Digit Recognition Workshop

### PR DESCRIPTION
**Pull request by rpalakkal**
i'll do the recording sometime soon!

### CODE REVIEW

File: workshops/digit_recognition/KerasMNIST.md [hunk 1]
```diff
@@ -0,0 +1,527 @@
...
+![examples of correct predictions](https://cloud-lc775uvcv.vercel.app/3kerasmnist_9_0.png)
+
+
+
+```py
+print(X_train[0])
+print("Shape: " + str(X_train[0].shape))
+```
+
+    [[  0   0   0   0   0   0   0   0   0   0   0   0   0   0   0   0   0   0
```
Comment (faisalsayed10) [inline workshops/digit_recognition/KerasMNIST.md:80]: can you please tell something about what this is? (like a one line description of this block)

**Review by faisalsayed10 — CHANGES_REQUESTED** [review]
Hello! Just did a review of this workshop, I personally liked this workshop! Just one thing that need to done:

- I feel that this workshop is a bit complex one, so maybe add a prerequisites section before hopping into the workshop?

**Comment by faisalsayed10** [comment]
Also, maybe rename the file to `README.md` because that's how all the workshops are structured.

**Comment by sampoder** [comment]
Closing for now, excited for the resubmission 🚀

**Pull request closed without merging**